### DISCUSSION TITLE
Bimonthly option in invoices

### DISCUSSION THREAD

**Discussion by CharlyJr** [firefly-iii/firefly-iii#5910] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

Hello! First of all I want to thank James for Firefly, and everyone who contributes to the project!!!

Will it be possible to add the bimonthly option (every two months) in invoices?

### Solution

Add Bimonthly option (every two months) in invoices?

**Comment by JC5** [comment]
Set it to "monthly", and set Skip to 1.

**Comment by CharlyJr** [comment]
Thank yo JC! This is Recurring Transactions... Will it be reflected in the Dashboard in invoices payable every two months, or will it be monthly?

**Reply by JC5** [reply]
Bi-monthly.
